virtcontainers/qemu_test.go

```diff
@@ -280,22 +280,34 @@ func TestQemuAddDeviceSerialPortDev(t *testing.T) {
 }
 
 func TestQemuAddDeviceKataVSOCK(t *testing.T) {
+	assert := assert.New(t)
+
+	dir, err := ioutil.TempDir("", "")
+	assert.NoError(err)
+	defer os.RemoveAll(dir)
+
+	vsockFilename := filepath.Join(dir, "vsock")
+
 	contextID := uint64(3)
 	port := uint32(1024)
-	vHostFD := os.NewFile(1, "vsock")
+	assert.NoError(err)
```

Comment:
this assert on err looks redundant - am I missing something subtle maybe?

Reply:
Good catch - a vim bounce issue I think ;) Updated.